tempodb/encoding/vparquet4/block_traceql.go

```diff
@@ -1364,27 +1369,38 @@ func (i *mergeSpansetIterator) Close() {
 //
 // Diagram:
 //
-//  Span attribute iterator: key    -----------------------------
-//                           ...    --------------------------  |
-//  Span attribute iterator: valueN ----------------------|  |  |
-//                                                        |  |  |
-//                                                        V  V  V
-//                                                     -------------
-//                                                     | attribute |
-//                                                     | collector |
-//                                                     -------------
-//                                                            |
-//                                                            | List of attributes
-//                                                            |
-//                                                            |
-//  Span column iterator 1    ---------------------------     |
-//                      ...   ------------------------  |     |
-//  Span column iterator N    ---------------------  |  |     |
-//    (ex: name, status)                          |  |  |     |
-//                                                V  V  V     V
-//                                            ------------------
-//                                            | span collector |
-//                                            ------------------
+//  Event attribute iterator: key    --------------------------------------------
+//                                                                            | |
+//  Event attribute iterator: valunN ---------------------------------------  | |
+//    											                            | | |
```

Comment:
Nice doc update!!!

Minor formatting improvements:
```suggestion
//                            ...   ------------------------------------------- |
//  Event attribute iterator: valueN ---------------------------------------  | |
//                                                                          | | |
```